src/k8s-engine/lima.ts

```diff
@@ -476,12 +507,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     });
 
     if (os.platform() === 'darwin') {
-      config.networks = [
-        {
-          lima:      'bridged',
-          interface: 'rd0',
-        },
-      ];
+      const hostNetwork = (await this.getDarwinHostNetworks()).find(n => n.IPv4 && n.dhcp);
```

Comment:
Not sure if testing for `n.IPv4` is useful; that field seems always there, even if it just has a `ConfigMethod` field.

Maybe we should test for `n.IPv4.Addresses` to make sure they got at least an address? And or `AdditionalRoutes`?

Also, will `find` return just a single entry, even when there are multiple matches?

Reply:
Yes, [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) returns the first result, or `undefined`; [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) returns an array of (possibly multiple, possibly zero) matches.